test/e2e/storage/drivers/csi.go

```diff
@@ -602,6 +607,36 @@ func (g *gcePDCSIDriver) PrepareTest(f *framework.Framework) (*testsuites.PerTes
 	}, cleanupFunc
 }
 
+// WaitForPDCSIController waits for all replicas in the deployment at driverNamespace/name to be ready.
+func WaitForPDCSIController(driverNamespace *v1.Namespace, name string, cs clientset.Interface) error {
```

Comment:
Oh if you want to export this, then maybe https://github.com/kubernetes/kubernetes/blob/master/test/e2e/framework/deployment/wait.go would be a better place for such a util

Reply:
I was looking at your https://github.com/kubernetes/kubernetes/pull/93120 and trying to do the same thing.

I guess this function as written is general, but I can see in the future adding pd-specific stuff to it. LMK

Reply:
I would prefer to keep exported functions from k/k as provider-unspecific as possible.